scripts/valhalla_build_config

```diff
@@ -264,7 +264,8 @@ config = {
     'max_reachability': 100,
     'max_radius': 200,
     'max_timedep_distance': 500000,
-    'max_alternates': 2
+    'max_alternates': 2,
+    'max_avoid_polygons_length': 10
```

Comment:
The real driver for performance here is how many bins are intersected. With this max circumference one can intersect around max 10 bins (depending on latitude), which (in urban environments) adds a good 2-3 seconds to the request. Assuming it's a single polygon. One can also do 20 mini polygons each intersecting another bin. I don't know what you think, maybe we should add a max param for amount of polygons passed?

Reply:
in other files, i see this value defaulting to `100` can we make them all match, it creates less surprises